### PR TITLE
added question to HTTPS Section

### PR DESCRIPTION
**Pull request by RaviTejaCMS**
Added Question "Can I browse normal https sites with Tor?" to HTTPS Section
Issue: https://dip.torproject.org/web/support/issues/35

### CODE REVIEW

**Comment by aadibajpai** [comment]
Hi! For starters, it's Tor and not TOR :)

**Comment by RaviTejaCMS** [comment]
> Hi! For starters, it's Tor and not TOR :)

Hey Thanks.!😉

**Comment by gusgustavo** [comment]
Hi, we should improve a little bit more here, because it's a common question that we have on our trainings. 

For example:

* [ ] What exactly HTTPS means, because other users that don't know what's it, will read and need to understand.
* [ ] Link to the TB-Manual: https://tb-manual.torproject.org/secure-connections/
* [ ] Add that Tor Browser have HTTPS-Everywhere and what exactly this plugin do.

**Comment by RaviTejaCMS** [comment]
Will update that😁

File: content/https/https-2/contents.lr [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+---
+title: Can I browse normal https sites with Tor?
+---
+seo_slug: browse-normal-https-sites
+---
+description:
+Short Answer, **Yes you can browse normal HTTPS Sites using Tor.**
+
+HTTPS Connections is used for secure communication over a computer network.You can read more about HTTPS [here](https://tb-manual.torproject.org/secure-connections/)
+Tor browser have [HTTPS Everywhere](https://www.eff.org/https-everywhere) plugin which automatically switches thousands of sites from insecure "http" to secure "https".
```
Comment (aadibajpai) [inline content/https/https-2/contents.lr]:
```suggestion
Tor Browser has the [HTTPS Everywhere](https://www.eff.org/https-everywhere) plugin which automatically switches thousands of sites from insecure "http" to secure "https".
```
I feel the grammar could be improved a bit :)

Comment (RaviTejaCMS) [reply content/https/https-2/contents.lr]: yeah missed it.!🧐

**Pull request merged**